src/tincd.c

```diff
@@ -322,6 +323,27 @@ static bool parse_options(int argc, char **argv) {
 	return false;
 }
 
+static sandbox_level_t read_sandbox_level(void) {
+	char *value = NULL;
+	sandbox_level_t level = SANDBOX_NORMAL;
+
+	if(get_config_string(lookup_config(&config_tree, "Sandbox"), &value)) {
+		if(!strcasecmp("off", value)) {
+			level = SANDBOX_NONE;
+		} else if(!strcasecmp("normal", value)) {
+			level = SANDBOX_NORMAL;
+		} else if(!strcasecmp("high", value)) {
+			level = SANDBOX_HIGH;
+		} else {
+			logger(DEBUG_ALWAYS, LOG_ERR, "Bad sandbox value %s, using default", value);
```

Comment:
That's bad. Consider making a typo, thinking Sandbox level is high, but tinc just using normal level. Yes there is a log message, but it's easy to miss. Another issue is if Sandbox level is set explicitly on a platform that we don't have sandboxing implemented for. It would be safer to exit tinc immediately if an unsupported or unknown value for Sandbox is used.